airbyte-integrations/connectors/source-postgres/src/testFixtures/java/io/airbyte/integrations/source/postgres/PostgresTestDatabase.java

```diff
@@ -19,10 +19,11 @@
 public class PostgresTestDatabase extends
     TestDatabase<PostgreSQLContainer<?>, PostgresTestDatabase, PostgresTestDatabase.PostgresConfigBuilder> {
 
-  public static enum BaseImage {
```

Comment:
why are you making this non static ?

Reply:
Intellij auto fix, nested static is redundant is for enum.

Reply:
yeah, I didn't realize that it was redundant.